enterprise/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/ide/WildflyStartRunnable.java

```diff
@@ -201,6 +201,12 @@ private void setupEnvironment(final InstanceProperties ip, Environment env) {
             javaOptsBuilder.append(" -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true");
         }
 
+        if(ip.getProperty(WildflyPluginProperties.PROPERTY_PORT_OFFSET) != null ) {
```

Comment:
@ehsavoie does this mean, that if a user enters the valid port offset `0` it will be ignored?
If so, the default from the config will be used and the server not connected to NB.